packages/fxa-auth-server/lib/senders/emails/partials/icon/index.mjml

```diff
@@ -0,0 +1,18 @@
+<mj-section css-class="mb-8">
```

Comment:
Noiiice this is a solid improvement. 👍

We could use the license at the top of this file.